hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/dog/AdditionalStudiesTransformations.scala

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.monster.dap.dog
+
+import org.broadinstitute.monster.dap.RawRecord
+import org.broadinstitute.monster.dogaging.jadeschema.fragment.HlesDogFutureStudies
+
+object AdditionalStudiesTransformations {
+
+  /**
+    * Parse all future-studies health fields out of a raw RedCap record,
+    * injecting them into a partially-modeled dog record.
+    */
+  def mapFutureStudies(rawRecord: RawRecord): HlesDogFutureStudies = {
```

Comment:
You could go one step further and merge the body of the function below into this one. I think the HlesDogBehavior object is structured that way.

Reply:
Thanks! I was using a more complicated transformation piece as a skeleton to build out this one but since this one is so simple it makes sense to condense it down to a smaller form.